introduction/payment-driver-getting-started.md

```diff
@@ -0,0 +1,100 @@
+# Getting started to write payment driver for Yagna
+
+This document aims to provide guidelines to develop a payment driver based on boilerplate code from [Yagna repository](https://github.com/golemfactory/yagna). It assumes reader has some experience in [Rust language](https://www.rust-lang.org/) and also Yagna project's codebase. We're providing an information for payment driver that will be integrated into Yagna daemon codebase and gains from reusing a lot of already provided components. It's also possible to develop stand-alone driver application that communicates with the Yagna daemon via TCP, but it won't be described here.
+
+
+## Overview
+
+Payment service of Yagna daemon expects that payment driver will handle all communication with particular L2 network provider (but it's not limited to Ethereum ecosystem).
+You might assume that payment driver should:
+
+- communicate with daemon services over Golem Service Bus (GSB),
+- provide operation like deposits, exits, transfer, account unlocking,
+- delegate transaction's signing to the Identity service,
+- notify Payment service about committed transactions,
+- retry operations that failed to due non-critical causes.
+
+Here is a checklist that driver implementors would want to follow:
+
+- [] Fork the Yagna repository and clone it locally to start working on the code,
```

Comment:
I'd add something like "create a new crate for your driver in `core/payment-drivers` directory.

Reply:
Bump.

Reply:
:) just in the separate (atomic) commit